modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -2375,6 +2375,52 @@ m|+++false+++
 |===
 
 
+[role=label--new-5.21]
+[[config_db.index.fulltext.eventually_consistent_apply_parallelism]]
+=== `db.index.fulltext.eventually_consistent_apply_parallelism`
+
+.db.index.fulltext.eventually_consistent_apply_parallelism
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|The number of threads that are applying the queued index updates for eventually consistent full-text indexes.
+|Valid values
+a|An integer that is minimum 1.
+|Default value
+m|+++1+++
+|===
+
+[role=label--new-5.21]
+[[config_db.index.fulltext.eventually_consistent_refresh_interval]]
+=== `db.index.fulltext.eventually_consistent_refresh_interval`
+
+.db.index.fulltext.eventually_consistent_refresh_interval
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|How often an eventually consistent full-text index will be refreshed (readers guaranteed to see changes). If set to `0` then refresh is done by the threads applying eventually consistent fulltext index updates.
+|Valid values
+a|A duration (Valid units are: ns, μs, ms, s, m, h and d; default unit is s).
+|Default value
+m|+++0s+++
+|===
+
+[role=label--new-5.21]
+[[config_db.index.fulltext.eventually_consistent_refresh_parallelism]]
+=== `db.index.fulltext.eventually_consistent_refresh_parallelism`
+
+.db.index.fulltext.eventually_consistent_refresh_parallelism
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|The number of threads that can do full-text index refresh in parallel, basically how many eventually consistent full-text indexes can be refreshed in parallel.
```

Comment:
```suggestion
a|The number of threads that can do full-text index refresh in parallel, i.e. the number of eventually consistent full-text indexes that can be refreshed in parallel.
```

Reply:
Perhaps we can just use the second sentence.

Same here - needs to be merged in the codebase before the cut-off.